### PR TITLE
Attempt at fixing issue with cross domain scripting

### PR DESCRIPTION
**Pull request by GitHK**
This is a request from  @odeimaiz to fix an issue users are having with jupyters not opening on Safari and Firefox.

See more https://github.com/jupyterhub/jupyterhub/issues/3117#issuecomment-857589466

### CODE REVIEW

**Review by odeimaiz — APPROVED** [review]
Thanks

File: docker/boot_notebook.bash [hunk 1]
```diff
@@ -54,6 +54,18 @@ cat > .jupyter_config.json <<EOF
 }
 EOF
 
+# cookie security options for cross domain
+cat > "$HOME/.jupyter/jupyter_notebook_config.py" <<EOF
+c.JupyterHub.tornado_settings = {
```
Comment (pcrespov) [inline docker/boot_notebook.bash:59]: `c.JupyterHub.tornado_settings` is repeated ... I guess is a bad typo?

File: docker/boot_notebook.bash [hunk 2]
```diff
@@ -54,6 +54,18 @@ cat > .jupyter_config.json <<EOF
...
+    'cookie_options': {'SameSite': 'Strict', 'Secure': True}
+}
+
+c.NotebookApp.tornado_settings = {
+    'cookie_options': {'SameSite': 'Strict', 'Secure': True}
+}
+c.NotebookApp.disable_check_xsrf = True
```
Comment (pcrespov) [inline docker/boot_notebook.bash:66]: Can you add a link to the doc of these options? This way it is simple to understand their meaning and allows easy access to doc to tune them in the future

**Review by pcrespov — CHANGES_REQUESTED** [review]

File: docker/boot_notebook.bash [hunk 3]
```diff
@@ -54,6 +54,18 @@ cat > .jupyter_config.json <<EOF
 }
 EOF
 
+# cookie security options for cross domain
+cat > "$HOME/.jupyter/jupyter_notebook_config.py" <<EOF
+c.JupyterHub.tornado_settings = {
+    'cookie_options': {'SameSite': 'Strict', 'Secure': True}
```
Comment (odeimaiz) [inline docker/boot_notebook.bash:60]: SameSite None, oder? same goes for the NotebookApp.tornado_settings

File: docker/boot_notebook.bash (hunk 1 above)
Comment (GitHK) [reply docker/boot_notebook.bash:59]: It is not repeated, it is different.

File: docker/boot_notebook.bash (hunk 2 above)
Comment (GitHK) [reply docker/boot_notebook.bash:66]: done

**Review by elisabettai — APPROVED** [review]

**Pull request merged**